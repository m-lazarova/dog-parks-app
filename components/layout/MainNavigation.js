import Link from 'next/link';
import classes from './MainNavigation.module.css';

const MainNavigation = () => {
  return (
    <header className={classes.header}>
      <div>
        <Link href='/'>The React Dog Parks</Link>
      </div>
      <nav>
        <ul>
          <li>
            <Link href='/all-dog-parks'>All Dog Parks</Link>
          </li>
          <li>
            <Link href='/new-dog-park'>Add a New Dog Park</Link>
          </li>
        </ul>
      </nav>
    </header>
  )
};

export default MainNavigation;