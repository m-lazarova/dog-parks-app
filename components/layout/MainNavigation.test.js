import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/react';
import MainNavigation from './MainNavigation';

describe('MainNavigation', () => {
  it('should render text "The React Dog Parks"', () => {
    render(<MainNavigation />);

    const dogParksString = screen.getByText('The React Dog Parks');
    expect(dogParksString).toBeInTheDocument();
  });

  it('should render text "All Dog Parks"', () => {
    render(<MainNavigation />);

    const dogParksString = screen.getByText('All Dog Parks');
    expect(dogParksString).toBeInTheDocument();
  });

  it('should render text "Add a New Dog Park"', () => {
    render(<MainNavigation />);

    const dogParksString = screen.getByText('Add a New Dog Park');
    expect(dogParksString).toBeInTheDocument();
  });
});