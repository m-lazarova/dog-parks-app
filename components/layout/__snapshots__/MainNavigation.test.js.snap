// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainNavigation Links render correctly 1`] = `
Array [
  <a
    href="/"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    The React Dog Parks
  </a>,
  <a
    href="/all-dog-parks"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    All Dog Parks
  </a>,
  <a
    href="/new-dog-park"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    Add a New Dog Park
  </a>,
]
`;
